fv/test-workload/test-workload.go

```diff
@@ -155,56 +155,66 @@ func main() {
 				// Without these sysctls enabled, interfaces will come up but they won't get a link local IPv6 address,
 				// which is required to add the default IPv6 route.
 				if err = writeProcSys("/proc/sys/net/ipv6/conf/all/disable_ipv6", "0"); err != nil {
+					log.WithError(err).Error("Failed to disable IPV6.")
```

Comment:
this is enabling IPv6; it's setting the disable flag to 0

Reply:
Fixed